distribution/docs/src/main/resources/_contents/_catalog-ui-contents/managing-catalog-ui-contents.adoc

```diff
@@ -0,0 +1,39 @@
+
+The Catalog UI application provides a metadata search and discovery, resource retrieval and workspace management with a 3d or optional 2d map visualization.
+
+=== ${catalog-ui} Prerequisites
+
+To use the ${catalog-ui} application, the following applications/features must be installed:
+
+* ${ddf-platform}
```

Comment:
For another ticket, but we may need to look at updating these app names to align with what the admins see now. That basically is remove `DDF` in `DDF Catalog`